string name
bool display_name
geometry_msgs/PolygonStamped polygon
---
